core/spec/helpers/base_helper_spec.rb

```diff
@@ -183,7 +183,9 @@ def link_to_tracking_html(options = {})
     context 'when try_spree_current_user is undefined' do
       let(:current_currency) { 'USD' }
 
-      before { I18n.locale = I18n.default_locale }
+      before do
+        allow(I18n).to receive(:locale) { I18n.default_locale }
```

Comment:
We could probably stick to the `and_return(I18n.default_locale)` syntax like everywhere else.  I think RSpec explicitly converts the block to a proc which will also be slower.

On a different note, those changes could have probably been a separate PR. Seems arbitrary to include them here

Reply:
@nciemniak do you need my branch for testing your change, or could we merge it without it, and then push updated i18n mocks to a separate branch.

Reply:
Yeah lets merge without the changes to tests, agree.